### DISCUSSION TITLE
How can I get the message I'm sending back?

### DISCUSSION THREAD

**Discussion by ghost** [Mateo-tem/discord-modals#42] [Q&A] [answered]
I want to delete his message after the bot response. I tried to do this, but it didn't work out.
`modal.reply("text").then(msg => msg.delete({ timeout: 3000 }));`
 How can I get a message to delete it?

**Comment by Mateo-tem** [comment, marked as answer]
You can use the `.deleteReply()` method:

```javascript
modal.reply("text");

setTimeout(() => {
  modal.deleteReply();
}, 3000);
```

**Reply by ghost** [reply]
Oh thank you! I'll try later
